db/seeds.rb

```diff
@@ -1,7 +1,416 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+t1 = Teacher.create(first_name: "Maryl", last_name: "Wealthall", email: "mwealthall0@jigsy.com", password: "password")
```

Comment:
I hope you didn't do this by hand

Reply:
I have a strategy (though it needs to be perfected..)